src/explorer.api/Explorers/RealColumnExplorer.cs

```diff
@@ -113,7 +113,7 @@ select new
             PublishMetric(new UntypedMetric(name: "histogram_buckets", metric: histogramBuckets));
 
             // Estimate Median
-            var processed = 0;
+            long processed = 0;
```

Comment:
You can use here `var processed = 0L;`

Reply:
I have another branch with small changes, i'll integrate this change there.